common/app/common/commercial/hosted/HostedGalleryPage.scala

```diff
@@ -13,7 +13,7 @@ case class HostedGalleryPage(
   ctaText: String,
   ctaLink: String,
   ctaButtonText: String,
-  ctaIndex: Integer,
+  ctaIndex: Integer = 0,
```

Comment:
Would be better to make this an `Option[Int]` if it's not always known at create time.